CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.9.0-beta+2
+
+> NOTE: This code is considered production quality, but depends on angular:
+> 5.0.0-beta+2 (5.0.0-beta+2 is used in production Google apps).
+
+### Breaking Changes
+* Remove `SelectableChangeNotifier` and deprecate `SelectableWithComposition`
+  with the intention to remove. They are widely unused, complicated the
+  implementations, and unsound in Dart 2.
+
+#### Material Dialog
+* Fix bug where a disposable could be added to its disposer after it had already
+  been destroyed.
+
+#### Material Icon
+* Add aria label and use them for trailing/leading icons of Material Input.
```

Comment:
At some point we should make a global search and replace in this file (and the API docs) of: /\baria\b/ -> ARIA